src/extender.rs

```diff
@@ -168,28 +168,28 @@ impl<H: HashT, NI: NodeIdT> Extender<H, NI> {
     fn vote_and_decision(
         &self,
         candidate_hash: &H,
-        voter_hash: &H,
+        member_hash: &H,
         candidate_creator: NodeIndex,
         candidate_round: Round,
     ) -> (bool, Option<bool>) {
         // Outputs the vote and decision of a unit u, computed based on the votes of its parents
         // It is thus required the votes of the parents to be up-to-date (if relative_round>=2).
-        let voter = self.units.get(voter_hash).unwrap();
-        if voter.round <= candidate_round {
+        let member = self.units.get(member_hash).unwrap();
+        if member.round <= candidate_round {
```

Comment:
Renaming votes -> member in this file is not that great of an idea :) "voter" here is a unit, that votes for some other "candidate" unit below it. I'm not saying this is the best naming convention, but it makes sense, and definitely voter != node/validator/member in this context.

Reply:
Sure...you know what happened :P fixed :)